content/en/synthetics/browser_tests/advanced_options.md

```diff
@@ -6,20 +6,27 @@ aliases:
   - /synthetics/guide/browser-tests-switch-tabs/
   - /synthetics/guide/browser-test-self-maintenance/
 further_reading:
+- link: "https://www.datadoghq.com/blog/browser-tests/"
+  tag: "Blog"
+  text: "User experience monitoring with Synthetic browser tests"
 - link: "/synthetics/browser_tests/actions/"
   tag: "Documentation"
   text: "Learn more about Browser Test Steps"
 ---
 
 ## Overview
 
+This page describes how Synthetic browser tests are self maintained and the various advanced options available in the recording workflow. 
```

Comment:
is it a good practice to have the 1st sentence of a page saying "This page describes ..."

Reply:
Yep, it's a common practice across our doc set to basically introduce the topics in this doc page.

Reply:
How about something like this?

```suggestion
This page describes how Synthetic browser tests locate elements with the Datadog algorithm and the Datadog user locator system. 
```

Reply:
yup and if you think that doesn't bring any value we should probably drop it